Add ls to aliases

scaryrawr opened the pull request:
This updates `ls` to be handled by `eza`.

scaryrawr committed:
Add ls alias

ManuelEthon commented:
Would be great to merge this but I guess also the Readme wants to be updated with the "new" alias?

scaryrawr committed:
Add `ls` entry to README

scaryrawr committed:
fix: pass required depth value to --level for eza v0.23.5+

eza v0.23.5 migrated to clap for CLI parsing, which changed
--level from an optional flag to a required argument. Update the
lt alias and its extended variants to pass a default depth of 2,
matching the previous implicit default behavior.

scaryrawr committed:
Add AGENTS.md
